docs/UserGuide.md

```diff
@@ -335,31 +337,32 @@ Format: `edit INDEX [n/MEMBER_NAME] [hp/MEMBER_PHONE] [e/MEMBER_EMAIL] [a/MEMBER
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 
-* When editing allergens, the existing allergens of the member will be removed i.e adding of allergens is not cumulative.
-* You can remove all the member’s allergens by typing `ag/` without specifying any allergens after it.
+* When editing [allergens](#6-glossary), the existing [allergens](#6-glossary) of the member will be removed i.e adding of allergens is not cumulative.
```

Comment:
I dont think u needed to hyperlink every word HAHA but thank you looks v good a lot of effort to do that